src/deoptim.cpp

```diff
@@ -18,7 +18,7 @@ void devol(double VTR, double f_weight, double fcross, int i_bs_flag,
            arma::mat    & ta_popP, arma::mat    & ta_oldP, arma::mat    & ta_newP, arma::colvec & t_bestP, 
            arma::colvec & ta_popC, arma::colvec & ta_oldC, arma::colvec & ta_newC, double       & t_bestC,      
            arma::colvec & t_bestitP, arma::colvec & t_tmpP, arma::mat & d_pop, Rcpp::List & d_storepop, 
-           arma::mat & d_bestmemit, arma::colvec & d_bestvalit, int & i_iterations, double i_pPct, long & l_nfeval,
+           arma::mat & d_bestmemit, arma::colvec & d_bestvalit, int & i_iterations, double i_pPct, double d_c, long & l_nfeval,
```

Comment:
hmm, this is supposed to be part of a **list** variable in 'RcppExport.{cpp, R}'. Last time when I added `d_reltol` and `i_steptol` the RcppExport files were changed